tools/Azure.Mcp.Tools.Monitor/tests/Azure.Mcp.Tools.Monitor.LiveTests/MonitorCommandTests.cs

```diff
@@ -74,63 +74,53 @@ public async Task Should_list_monitor_workspaces()
         Assert.NotEmpty(array);
     }
 
-    [Fact(Skip = "Intermittent failures due to slow ingestion")]
+    [Fact]
     public async Task Should_get_table_contents()
     {
         // Query AzureMetrics table - fastest to propagate and most reliable
+        var workspace = "monitor-query-ws";
+        var resourceGroup = "static-test-resources";
```

Comment:
@weshaggard / @hallipr, is there a way to go about this that's more robust if these resources change locations?

Reply:
For static resources I don't know if there is a better way than just hard-coding them somewhere. They are not something that move often.